plugins/provisioners/docker/config.rb

```diff
@@ -4,9 +4,11 @@ module VagrantPlugins
   module DockerProvisioner
     class Config < Vagrant.plugin("2", :config)
       attr_reader :images
+      attr_accessor :post_install_provisioner
 
       def initialize
         @images = Set.new
+        @post_install_provisioner = []
```

Comment:
Should either remove the `@post_install_provisioner` initialization or just set it to `nil`